esp/templates/program/modules/programprintables/options.html

```diff
@@ -87,23 +87,23 @@ <h3>Student Lists</h3>
 {% if program|hasModule:"TeacherModeratorModule" %}
     <li>Class Rosters</li>
     <ul>
-        <li><a href="./classrosters" title="Class Rosters">By teachers</a></li>
-        <li><a href="./classrostersbymoderator" title="Class Rosters">By {{ program.getModeratorTitle|lower }}s</a></li>
+        <li><a href="./classrosters?recipient_type=Teacher&base_list=class_approved" title="Class Rosters">By teachers</a></li>
+        <li><a href="./classrostersbymoderator?recipient_type=Teacher&base_list=assigned_moderator" title="Class Rosters">By {{ program.getModeratorTitle|lower }}s</a></li>
     </ul>
     <li>Attendance Sheets</li>
     <ul>
-        <li><a href="./classrosters/attendance" title="Attendance Sheets">By teachers</a></li>
-        <li><a href="./classrostersbymoderator/attendance" title="Attendance Sheets">By {{ program.getModeratorTitle|lower }}s</a></li>
+        <li><a href="./classrosters/attendance?recipient_type=Teacher&base_list=class_approved" title="Attendance Sheets">By teachers</a></li>
+        <li><a href="./classrostersbymoderator/attendance?recipient_type=Teacher&base_list=assigned_moderator" title="Attendance Sheets">By {{ program.getModeratorTitle|lower }}s</a></li>
     </ul>
 {% else %}
-    <li><a href="./classrosters" title="Class Rosters">Class Rosters</a> (by teachers)</li>
-    <li><a href="./classrosters/attendance" title="Attendance Sheets">Attendance Sheets</a> (by teachers)</li>
+    <li><a href="./classrosters?recipient_type=Teacher&base_list=class_approved" title="Class Rosters">Class Rosters</a> (by teachers)</li>
+    <li><a href="./classrosters/attendance?recipient_type=Teacher&base_list=class_approved" title="Attendance Sheets">Attendance Sheets</a> (by teachers)</li>
 {% endif %}
 <li><a href="./classchecklists" title="Class Checlists">Class Checklists</a> (all confirmed students)</li>
-<li><a href="./studentchecklist" title="Student Checklist">Student Checklist</a> (by students)</li>
-<li><a href="./student_financial_spreadsheet">Student Financial Spreadsheet</a> (used by the check-scanning app)</li>
-<li><a href="./studentsbyname" title="Student List">Students by Name</a> (by students)</li>
-<li><a href="./emergencycontacts" title="Student List">Students' Emergency Contact Info</a> (by students)</li>
+<li><a href="./studentchecklist?recipient_type=Student&base_list=enrolled" title="Student Checklist">Student Checklist</a> (by students)</li>
+<li><a href="./student_financial_spreadsheet?recipient_type=Student&base_list=enrolled">Student Financial Spreadsheet</a> (used by the check-scanning app)</li>
+<li><a href="./studentsbyname?recipient_type=Student&base_list=enrolled" title="Student List">Students by Name</a> (by students)</li>
+<li><a href="./emergencycontacts?recipient_type=Student&base_list=enrolled" title="Student List">Students' Emergency Contact Info</a> (by students)</li>
 {% if not li_types|length_is:0 %}
 <li>Students who have ordered/reserved line items:
```

Comment:
I think these could use default GET parameters too, although a bit more complicated

Reply:
I'm just gonna do "All students in database", because I guess there might be students that ordered something that aren't enrolled?